Tests/UtilityTests/ProgressBarTests.swift

```diff
@@ -88,10 +52,10 @@ final class ProgressBarTests: XCTestCase {
         }
         thread.start()
         runProgressBar(bar)
-        pty.closeSlave()
         // Make sure to read the complete output before checking it.
+        pty.close()
```

Comment:
We shouldn't close these fds at the same time. We should keep the separate closeSlave() and closeMaster() methods and change close() method to call those method in succession.

Reply:
Oh I see. Fixed!